### PR TITLE
Update core_mmu_v7.c

### PR DESCRIPTION
**Pull request by xiaoqiangdu**
Fix the problem that some page entries can not be mapped in function map_page_memarea

### CODE REVIEW

File: core/arch/arm32/mm/core_mmu_v7.c [hunk 1]
```diff
@@ -544,7 +544,7 @@ static paddr_t map_page_memarea(struct tee_mmap_region *mm)
 	}
 
 	/* Fill in the entries */
-	while ((pg_idx * SMALL_PAGE_SIZE) < mm->size) {
+	while ((pg_idx * SMALL_PAGE_SIZE) < (mm->size + (mm->pa & SECTION_MASK))) {
```
Comment (jenswi-linaro) [inline core/arch/arm32/mm/core_mmu_v7.c]: Line is too long

**Comment by jenswi-linaro** [comment]
The commit message would be more uniform with the other commit messages if put like:
```
arm32 core_mmu_v7.c: bugfix map_page_memarea()

Fixes  the problem that some page entries can not be mapped in
map_page_memarea().

Signed-off-by: Xiaoqiang Du <xiaoqiang.du@linaro.org>
```

File: core/arch/arm32/mm/core_mmu_v7.c (hunk 1 above)
Comment (xiaoqiangdu) [inline core/arch/arm32/mm/core_mmu_v7.c]: I will split the code into two lines

Comment (xiaoqiangdu) [inline core/arch/arm32/mm/core_mmu_v7.c]: while ((pg_idx * SMALL_PAGE_SIZE) < 
          (mm->size + (mm->pa & SECTION_MASK))) {

**Comment by pascal-brand-st-dev** [comment]
Travis errors to be fixed.

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>
    Tested-by: Pascal Brand <pascal.brand@linaro.org> (STM platform)

**Comment by pascal-brand-st-dev** [comment]
Please check section https://github.com/OP-TEE/optee_os/blob/master/documentation/github.md#finalizing-the-contribution so that we push your contribution on master.

**Comment by xiaoqiangdu** [comment]
ok, I will do it

**Comment by pascal-brand-st-dev** [comment]
@xiaoqiangdu Just let me know if you need some help to finalize the pull-request.

**Comment by xiaoqiangdu** [comment]
I follow the
https://github.com/OP-TEE/optee_os/blob/master/documentation/github.md#finalizing-the-contribution
to finalize the contributing, but failed. Now I am trying to do it again.

**Comment by pascal-brand-st-dev** [comment]
@xiaoqiangdu Did you try again?

**Comment by pascal-brand-st** [comment]
Merged manually

**Pull request closed without merging**